components/_patterns/02-molecules/about-x/about-x.twig

```diff
@@ -0,0 +1,26 @@
+{% set about_x__base_class = 'about-x' %}
+
+<div {{ bem(about_x__base_class) }}>
+  {% include "@atoms/02-text/00-headings/_heading.twig" with {
+    heading_base_class: 'heading',
+    heading_blockname: about_x__base_class,
+    heading_level: '2',
+    heading: about_x__heading,
+  } %}
+  <div {{ bem('content', [], about_x__base_class) }}>
+    {% if about_x__image %}
+      <div {{ bem('image', [], about_x__base_class) }}>
+        {% block about_x__image %}
+          {% include "@atoms/04-images/00-image/responsive-image.twig" with {
+            output_image_tag: true,
+            img_src: about_x__image_src,
+          } %}
+        {% endblock %}
+      </div>
+    {% endif %}
+    {# This should be updated to use the text atom once it's merged in from the other PR #}
+    <div {{ bem('text', [], about_x__base_class) }}>
+      {{ about_x__content }}
+    </div>
+  </div>
+</div>
```

Comment:
Newline error

Sorry if these review bits are annoying! They're super easy to see on Github, but not as obvious in the editor for sure